libexec/bats-core/bats

```diff
@@ -7,7 +7,13 @@ version() {
 
 usage() {
   version
-  echo "Usage: bats [-c] [-r] [-p | -t] <test> [<test> ...]"
+  printf "Usage: bats [-c] [-r] [-p | -t] <test> [<test> ...]\n"
```

Comment:
why the printf here instead of echo?

Reply:
Preference based on the general notion that `echo` isn't standardized and `printf` is more consistent (even though both are Bash builtins). You can do a lot more cool stuff with `printf`, too, so I like to promote its awareness.

Some fun reading on the subject: https://unix.stackexchange.com/a/65819

Reply:
I'm cool with it, but we should probably move all echos to printfs, no? Instead of here and there?

Reply:
Done in #120.